elide-integration-tests/src/test/java/com/yahoo/elide/tests/ResourceIT.java

```diff
@@ -737,8 +737,8 @@ public void testForbiddenDeleteEmptyCollectionRelationship() throws Exception {
 
     @Test(priority = 11)
     public void testDeleteParent() {
+        //Should work without content-type and body
         given()
```

Comment:
I think this may still send a body. Does this test case blow up without your new method?